ogr/services/github.py

```diff
@@ -161,6 +166,78 @@ def get_fork(self, create: bool = True) -> Optional["GithubProject"]:
                 return None
         return self._construct_fork_project()
 
+    def get_issue_list(self, status: IssueStatus = IssueStatus.open) -> List[Issue]:
+        issues = self.github_repo.get_issues(
+            state=status.name, sort="updated", direction="desc"
+        )
+        try:
+            return [self._issue_from_github_object(issue) for issue in issues]
+        except UnknownObjectException:
+            return []
+
+    def get_issue_info(self, issue_id: int) -> Issue:
+        issue = self.github_repo.get_issue(number=issue_id)
+        return self._issue_from_github_object(issue)
+
+    def _get_all_issue_comments(self, issue_id: int) -> List[IssueComment]:
+        issue = self.github_repo.get_pull(number=issue_id)
+        return [
+            self._issuecomment_from_github_object(raw_comment)
+            for raw_comment in issue.get_issue_comments()
+        ]
+
+    def issue_comment(
+            self,
+            issue_id: int,
+            body: str
+    ) -> IssueComment:
+        """
+        Create comment on an issue.
+
+        :param issue_id: int The ID of the issue
+        :param body: str The text of the comment
+        :return: IssueComment
+        """
+        github_issue = self.github_repo.get_issue(number=issue_id)
+        comment = github_issue.create_comment(body)
+        return self._issuecomment_from_github_object(comment)
+
+    def issue_open(
```

Comment:
I would choose a better name here. It might be confusing that `issue_open` returns if the issue is open. I would prefer `create_issue` here. 

Same thing for `issue_close`.

Reply:
I agree with changing `issue_open` to `create_issue`. However, `issue_close` is the same naming convention as other already existing functions e.g. `pr_close`.

Reply:
Yes, you are right, so maybe to keep API the same we could go for: `issue_create` and `issue_close` despite the fact that `issue_close` (and `pr_close`) might be confusing.

Reply:
The long-term plan is to get rid of the `issue_close`/`pr_close`/`pr_merge` methods and move them to `Issue`/`PullRequest` ( see #86 ). (`my_issue.close()`)

You can leave it as is for now...